networkx/algorithms/cluster.py

```diff
@@ -352,20 +339,21 @@ def square_clustering(G, nodes=None):
     if nodes is None:
         node_iter = G
     else:
-        node_iter =  G.nbunch_iter(nodes) 
+        node_iter =  G.nbunch_iter(nodes)
     clustering = {}
     for v in node_iter:
-        clustering[v] = 0.0
-        potential=0
-        for u,w in combinations(G[v], 2):
+        clustering[v] = 0
+        potential = 0
+        for u, w in combinations(G[v], 2):
             squares = len((set(G[u]) & set(G[w])) - set([v]))
             clustering[v] += squares
-            degm = squares + 1.0
+            degm = squares + 1
             if w in G[u]:
                 degm += 1
             potential += (len(G[u]) - degm) * (len(G[w]) - degm) + squares
         if potential > 0:
             clustering[v] /= potential
     if nodes in G: 
-        return list(clustering.values())[0] # return single value
+        # Return the value of the sole entry in the dictionary.
+        return arbitrary_element(clustering.values())
```

Comment:
I suppose `sole_element()` would read better. I could create it as an alias to `arbitrary_element`, and document that `sole_element()` has a pre-condition that its argument must have length one. In any case, `list(iterable)[0]` is less readable, I think, because the others indicate a bit more what's happening in plain English.